### ISSUE TITLE
How to implement bayesian optimization for hyperparameter tuning

### ISSUE THREAD

**Issue by Elizbellou** [dbolya/yolact#724]
@dbolya Hello there, like #246 issue, Im thinking implementing bayessian optimization in yolact with custom dataset, in an attempt to improve results. I would like to ask in which original code files should I make changes and if there is a code example in training notebook (Im using jupyter or colab). Thanks in advance for your answer.

**Comment by anthonygofin** [comment]
Hello Elizbellou. Did you find a way to implement bayesian optimization ? 
I am running on 15 Coco sub-classes and, by doing fine-tuning my model performs slitghly worse than the original model. I'd like to search the hyper-parameters to improve results.

 Thanks in advance for your answer.

**Comment by Elizbellou** [comment]
@anthonygofin unfortunately I did not implement bayesian opt. Can't help still. However, I tried grid search for fine tuning which time consuming and got me worse results than with default config or no improvement with some hyper-parameters. What got me slightly higher mAP eventually was changing the learning steps according to max iterations that  I set. And resnet50 fit better for my dataset. Aspect ratios for example did not have any effect in my case. Sorry for not helping.
